scss/_skeletons.scss

```diff
@@ -0,0 +1,60 @@
+// .skeleton-line      - the skeleton container
+// .skeleton-dash      - the "text" replacement
+
+.skeleton-line {
+  display: flex;
+  align-items: center;
+  width: 100%;
+
+  // We use a &nbsp; to preserve
+  // the font size of the parent
+  &::before {
+    width: 0;
+    content: "\00A0";
+  }
+
+  // Animation
+  &.skeleton-glow {
+    animation: skeleton-glow 2s ease-in-out infinite;
+  }
+}
+
+.skeleton-dash {
+  width: 100%;
+  height: 1em;
+  background-color: currentColor;
+
+  + .skeleton-dash {
+    margin-left: 4px;
+  }
+}
+
+// Sizing
+.skeleton-xs {
+  > .skeleton-dash {
+    height: .6em;
+  }
+}
+
+.skeleton-sm {
+  > .skeleton-dash {
+    height: .8em;
+  }
+}
+
+.skeleton-lg {
+  > .skeleton-dash {
+    height: 1.2em;
+  }
+}
+
+// The animation effect
+@keyframes skeleton-glow {
+  0%,
+  100% {
+    opacity: 1;
+  }
+  50% {
```

Comment:
No need to define `0%` and `100%` here since `opacity: 1` is the initial value. See an example: https://codepen.io/ffoodd/pen/qBNdMLY